quick-start/index.html

```diff
@@ -0,0 +1,166 @@
+
+<!-- *** COMPONENTS *** -->
+<div class="screen">
+    <h2>Components</h2>
+    <p>
+        Everything drawn in Pencil.js is a {Component}. They all share the same behavior.
+    </p>
+    <p>
+        You will always declare {Component} with a position followed a number of parameters. For example, {Circle} only need a position and a radius.
+    </p>
+    <div class="example" data-script="n3"></div>
+    <p>
+        Once a {Component} is added to a {Scene} or another {Component}, it will be displayed on screen.
+    </p>
+</div>
+
+<!-- *** OPTIONS *** -->
+<div class="screen">
+    <h2>Options</h2>
+    <p>
+        In order to control the look of our {Circle}, let's change its options and rename it to <code>sun</code> to match its new look.
+    </p>
+    <p>
+        Let's also change the {Scene} background color.
+        Since we don't want to change the first argument of the {Scene} constructor, we can leave it at <code>undefined</code>.
+    </p>
+    <div class="example" data-script="n4"></div>
+    <p>
+        In the same way, we can add the ocean with a {Rectangle}.
+    </p>
+    <div class="example" data-script="n5"></div>
+</div>
+
+<!-- *** INTERACTION *** -->
+<div class="screen">
+    <h2>Interaction</h2>
+    <p>
+        A lot of Pencil.js is build around interaction with the user so you can build beautiful and interesting composition.
+    </p>
+    <p>
+        Firstly, we need to replace the <code>render</code> function (which render the scene only once) to <code>startLoop</code>.
+        This will make the scene update itself.
+    </p>
+    <p>
+        Secondly, let's make the <code>sun</code> follow the mouse. For that, we'll listen to the <code>"draw"</code> event
+        and update the <code>sun</code>'s position.
+    </p>
+    <div class="example" data-script="n6"></div>
+    <p>
+        The <code>lerp</code> function, move the position by 5% of the distance each time, giving it a smoother movement.
+    </p>
+</div>
+
+<!-- *** COLOR *** -->
+<div class="screen">
+    <h2>Colors</h2>
+    <p>
+        The <code>sun</code> is setting, but the sky color stay the same.
+        We can improve this by changing the sky's color according to the <code>sun</code>'s position.
+    </p>
+    <div class="example" data-script="n7"></div>
+    <p>
+        Note that, like {Position}, {Color} also have a <code>lerp</code> function to go from one color to another.
+    </p>
+    <p>
+        Now, let's do the same for the <code>ocean</code>.
+        But instead of changing the color, let's add a gradient and move it around.
+    </p>
+    <div class="example" data-script="n8"></div>
+</div>
+
+<div class="screen">
+    <h2>Particles</h2>
+    <p>
+        Final touch ! Let's add stars when the sky turn dark.
+        We could add a bunch of {Star}, but this could have an impact on drawing performance.
+        Instead, let's use {Particles}. They are build to around performance and can draw thousands of the same item.
```

Comment:
"They are build to around performance"
Not sure what to do with this, but it needs to be corrected.

Reply:
What's bothering you ?
I just wanted to explain the usage of Particles here.

Reply:
Maybe I'm missing some rules of the English language to comprehend this first part of the sentence. I am tripping over "build to around".
"They are *build to around* performance"
* They are built to perform
* They are built around performance
* They are built with performance in mind
* They perform faster/quicker

Reply:
Lol, that was just a typo. I was going for "They are built around performance". What's the best solution according to you ?

Reply:
They perform better when you draw the same item a thousand times.